Adafruit_MPL3115A2.cpp

```diff
@@ -132,6 +132,22 @@ void Adafruit_MPL3115A2::setSeaPressure(float SLP) {
   i2c_dev->write(buffer, 3);
 }
 
+/*!
+ *  @brief Set the oversample rate
+ *  @param sampleRate oversample rate from 0 to 7
+ * The higher the oversample rate the more time between samples
+ */
+void Adafruit_MPL3115A2::setOversampleRate(int8_t sampleRate) {
```

Comment:
Allow passing in the oversample enum defined in the header file. For example, ` MPL3115A2_CTRL_REG1_OS32`.

Reply:
I don't understand how I would pass the oversample rate enum that are defined in the header file. If I understand what you are asking, I am not sure that it would be as clear to the user how to use the different values. The user would have to change between setOverSampleRate(MPL3115A2_CTRL_REG1_OS32) in their sketch and the other 24 character enum's to set the rate. Instead of just entering 0 through 7. But maybe I am not understanding what you are proposing.